### PR TITLE
Metaball billboard 

### PR DESCRIPTION
**Pull request by Zsupi**
Signed-off-by: Németh Botond <nemethbotond.sopron@gmail.com>

### CODE REVIEW

File: Framework/Mesh.cpp [hunk 1]
```diff
@@ -67,7 +67,9 @@ void Mesh::draw(std::shared_ptr<Camera> camera) {
 	glm::mat4 viewProjMatrix = P * V;
 
 	modelMatrix =  glm::translate(position) * glm::scale(scale);
-	material->setModelMatrix(modelMatrix).setViewProjMatrix(viewProjMatrix).setRayDirMatrix(Ray);
+	material->setModelMatrix(modelMatrix)
+		.setViewProjMatrix(viewProjMatrix)
+		.setRayDirMatrix(Ray);
```
Comment (Dolnay-Adam) [inline Framework/Mesh.cpp:72]: No need to set rayDirMatrix two times

File: Framework/SSBO.cpp [hunk 2]
```diff
@@ -22,6 +22,10 @@ void SSBO::Unbind()
 
 //Different functions for different kind of buffers(vec4, vec3 etc.)
 
+/*
+
+*/
```
Comment (Dolnay-Adam) [inline Framework/SSBO.cpp:27]: Empty comment is unnecessary

File: Framework/SSBO.h [hunk 3]
```diff
@@ -15,6 +15,7 @@ class SSBO
 	void Bind(int index);
 	void Unbind();
 	void LoadData(std::vector<glm::vec4> data, GLsizeiptr size);
+	void LoadData(std::vector<glm::vec3> data, GLsizeiptr size);
```
Comment (Dolnay-Adam) [inline Framework/SSBO.h:18]: Is this implemented?

**Review by Dolnay-Adam — APPROVED** [review]

**Review by Dolnay-Adam — APPROVED** [review]

**Comment by Zsupi** [comment]
From now on, we can use the GL version 4.3.

image

**Pull request merged**